src/ceph-volume/ceph_volume/devices/lvm/activate.py

```diff
@@ -262,11 +262,11 @@ def activate(self, args, osd_id=None, osd_fsid=None):
             logger.info('unable to find a journal associated with the OSD, assuming bluestore')
             return activate_bluestore(lvs,
                                       no_systemd=args.no_systemd,
-                                      no_tmpfs=args.no_tmpfs)
+                                      tmpfs=(not args.no_tmpfs))
         if args.bluestore:
             activate_bluestore(lvs,
                                no_systemd=args.no_systemd,
-                               no_tmpfs=args.no_tmpfs)
+                               tmpfs=(not args.no_tmpfs))
```

Comment:
We are seeing failures like following in ceph-ansible testing:

```
    Traceback (most recent call last):
      File "/sbin/ceph-volume", line 9, in <module>
        load_entry_point('ceph-volume==1.0.0', 'console_scripts', 'ceph-volume')()
      File "/usr/lib/python2.7/site-packages/ceph_volume/main.py", line 38, in __init__
        self.main(self.argv)
      File "/usr/lib/python2.7/site-packages/ceph_volume/decorators.py", line 59, in newfunc
        return f(*a, **kw)
      File "/usr/lib/python2.7/site-packages/ceph_volume/main.py", line 149, in main
        terminal.dispatch(self.mapper, subcommand_args)
      File "/usr/lib/python2.7/site-packages/ceph_volume/terminal.py", line 194, in dispatch
        instance.main()
      File "/usr/lib/python2.7/site-packages/ceph_volume/devices/lvm/main.py", line 40, in main
        terminal.dispatch(self.mapper, self.argv)
      File "/usr/lib/python2.7/site-packages/ceph_volume/terminal.py", line 194, in dispatch
        instance.main()
      File "/usr/lib/python2.7/site-packages/ceph_volume/devices/lvm/create.py", line 69, in main
        self.create(args)
      File "/usr/lib/python2.7/site-packages/ceph_volume/decorators.py", line 16, in is_root
        return func(*a, **kw)
      File "/usr/lib/python2.7/site-packages/ceph_volume/devices/lvm/create.py", line 32, in create
        Activate([]).activate(args)
      File "/usr/lib/python2.7/site-packages/ceph_volume/decorators.py", line 16, in is_root
        return func(*a, **kw)
      File "/usr/lib/python2.7/site-packages/ceph_volume/devices/lvm/activate.py", line 269, in activate
        tmpfs=(not args.no_tmpfs))
    AttributeError: 'Namespace' object has no attribute 'no_tmpfs'
```

https://2.jenkins.ceph.com/job/ceph-ansible-prs-dev-centos-non_container-lvm_osds/915/consoleFull#1550444531621117e0-1285-4893-a7ba-14c204302a78

Reply:
looks like ceph-volume jobs haven't been triggered on this PR

Reply:
Gah.. see #30788 to revert that change (which ends up not being needed)